### PR TITLE
WIP: Use only one icon theme for monochrome icons

### PR DESCRIPTION
**Pull request by TheOneRing**
This one is still a wip.
We where using a mix of different icon themes, I replaced the monochrome icons with icons from https://fontawesome.com/ which we where already using for the quit button.

We could also think on replacing the colored icons too.
I guess there are a lot of icon packs out there we could evaluate too
Please leave some feedback

image

### CODE REVIEW

**Comment by TheOneRing** [comment]
Looks better in black (would be white for dark mode)
image

**Comment by TheOneRing** [comment]
Screenshot 2020-08-20 181108




#B5B6BB

**Comment by TheOneRing** [comment]
photo_2020-08-20_19-07-45

File: resources/font-awesome/update_color.ps1 [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+if (-not $COLOR) {
+    Write-Host "Please specify color"
+    exit(1)
+}
+Get-ChildItem "*.svg" | % {
+    $tmp = Get-Content $_
+    $tmp = $tmp -replace "fill=`"\S+`"", "fill=`"$COLOR`""
+    Set-Content -Path $_ -Value $tmp
+
+}
```
Comment (dschmidt) [inline resources/font-awesome/update_color.ps1:12]: Can you add a help section to this script?

**Comment by TheOneRing** [comment]
Superseded by https://github.com/owncloud/client/pull/8038

**Pull request closed without merging**